src/main/scala/cromwell/binding/command/ParameterCommandPart.scala

```diff
@@ -13,7 +13,11 @@ case class ParameterCommandPart(wdlType: WdlType, name: String) extends CommandP
     if (wdlType != paramValue.wdlType) {
       throw new UnsupportedOperationException(s"Incompatible type for $name: need a $wdlType, got a ${paramValue.wdlType}")
     }
+
     /* TODO: this also is a gross over-simplification */
-    paramValue.value.toString
+    paramValue match {
```

Comment:
I gave this a little more thought, and I'm thinking that this is basically a stop-gap for now anyway.  I think in the near future we'd get rid of asString altogether, and instead use a serialization class.  I think I'll add a comment to this and we can get rid of asString later.